roles/containerd/defaults/main.yml

```diff
@@ -33,6 +33,9 @@ versions:
   1.25.6:
     containerd_version: "1.6.8"
     runc_version: "v1.1.4"
+  1.26.3:
+    containerd_version: "1.7.0"
+    runc_version: "v1.1.4"
```

Comment:
Is this the correct runc version?

Reply:
it was not, thanks! I've updated it and testing the upgrade process I noticed that runc wasn't being updated because the downloaded file for the previous version was still in /tmp and ansible skipped the download (of the new version). I added the checksum check so:
1. the download is triggered when the version changes
2. as side-effect we check that the file was correctly downloaded